lib/client.js

```diff
@@ -72,8 +72,8 @@ const CircuitBreaker = require('circuit-breaker-js');
  * @throws {ServiceClient.Error}
  * @returns {ServiceClient.Response}
  */
-const decodeResponse = (response) => {
-    const contentType = response.headers['content-type'] || 'application/json';
+const decodeResponse = (response, accept) => {
+    const contentType = response.headers['content-type'] || accept;
```

Comment:
This is not the correct way to treat `accept` header. I recommend to consult with [RFC2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.1) and probably the best way is to find a node module with a compliant parser and solver, or if none exists, write it yourself (but this will take a while).